app/models/manageiq/providers/ansible_tower/inventory/parser/automation_manager.rb

```diff
@@ -23,7 +23,7 @@ def configured_systems
       inventory_object.hostname = host.name
       inventory_object.virtual_instance_ref = host.instance_id
       inventory_object.inventory_root_group = persister.inventory_root_groups.lazy_find(host.inventory_id.to_s)
-      inventory_object.counterpart = persister.vms.lazy_find(host.instance_id)
+      inventory_object.counterpart = persister.cross_link_vms.lazy_find({:uid_ems => host.instance_id}, {:ref => :by_uid_ems})
```

Comment:
NOTE this was using the uid_ems as the manager_ref previously due to the vms collection definition in the automation persister: https://github.com/ManageIQ/manageiq/pull/21654/files#diff-7422cd6fb687662faa3ce1db090a4888c9ef4c71b952e08d94435bd115d0d6b4L47